src/util/versionstore.cpp

```diff
@@ -151,6 +152,10 @@ QStringList VersionStore::dependencyVersions() {
     sf_command(nullptr, SFC_GET_LIB_VERSION, sndfile_version, sizeof(sndfile_version));
     // Null-terminate just in case.
     sndfile_version[sizeof(sndfile_version) - 1] = '\0';
+
+    int eburMaj, eburMin, eburP = 0;
```

Comment:
this leaves the first two variables uninitialized, which is one of the key reason why this style is discouraged.
```suggestion
    int eburMaj = 0;
    int eburMin = 0; 
    int eburP = 0;
```

Reply:
thanks, I pushed a fixup.